File: burakcanyildizbackend/Models/Burs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace burakcanyildizbackend.Models
{
    public class Burs : BaseEntity
    {
        [Key]

        public int Id { get; set; }

        public int CompanyId { get; set; }

        public Company Company { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public int isShow { get; set; } = 0;

        public int Kontenjan { get; set; } = 1;
    }
}

File: burakcanyildiz/lib/Models/burs.dart
import 'package:burakcanyildiz/Models/company.dart';

class Burs {
  int id;
  int companyId;
  Company company;
  String name;
  String description;
  int isShow;
  int kontenjan;

  Burs({this.id, this.companyId, this.company, this.name, this.description,this.isShow,this.kontenjan});

  Burs.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    companyId = json['companyId'];
    company =
        json['company'] != null ? Company.fromJson(json['company']) : null;
    name = json['name'];
    description = json['description'];
    isShow = json['isShow'];
    kontenjan = json['kontenjan'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['companyId'] = this.companyId;
    data['company'] = this.company;
    data['name'] = this.name;
    data['description'] = this.description;
    data['isShow'] = this.isShow;
    data['kontenjan'] = this.kontenjan;
    return data;
  }
}
